src/js/utils/events.js

```diff
@@ -53,7 +53,7 @@ export function on(elem, type, fn){
           if (event.isImmediatePropagationStopped()) {
             break;
           } else {
-            handlersCopy[m].call(elem, event);
+            handlersCopy[m].apply(elem, [event].concat(evtData));
```

Comment:
@carpasse did you want to update these to match the single hash arg or are you happy with it as is? Other than this question I think it's good to pull in.